unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ChatController/IChannelsUtils.cs

```diff
@@ -0,0 +1,7 @@
+namespace DCL.Chat
+{
+    public interface IChannelsUtils
```

Comment:
i think there is a problem with the naming, this is more like a service not utils

Reply:
Done in:
- https://github.com/decentraland/unity-renderer/pull/3195/commits/e685128b1ca07b80d977c63975ac850bc135760a
- https://github.com/decentraland/unity-renderer/pull/3195/commits/fb639e2f5675f2c08b0e96c9eea302c688a2238b